src/status_im/accounts/screen.cljs

```diff
@@ -34,7 +34,9 @@
      [account-bage address photo-path name]]]])
 
 (defn create-account [_]
-  (dispatch [:reset-app #(dispatch [:navigate-to :chat console-chat-id])]))
+  (dispatch [:initialize-db])
```

Comment:
Might be nicer to have  a create-account event in the event namespace that does a dispatch-n on these events rather than a function in the views (also was renaming this ns out of the scope of this PR ?)